cpp/util/openssl_scoped_types.h

```diff
@@ -44,6 +44,7 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 
+namespace util {
```

Comment:
I reckon that if you put this in the `cert_trans` namespace, it would both make sense, and not need qualifying in most places. :wink: